src/mlpack/methods/ann/layer/policygradient.hpp

```diff
@@ -0,0 +1,95 @@
+/**
+ * @file advantage_loss.hpp
```

Comment:
@zoq , I changed the name of the file to make it generic. This describes a class of policy gradients (https://arxiv.org/pdf/1506.02438.pdf) (Page 2 :  PRELIMINARIES) .